pkg/tablewriter/tablewriter_test.go

```diff
@@ -49,8 +49,7 @@ func TestTableWriter(t *testing.T) {
 	}
 
 	buf := bytes.NewBuffer(nil)
```

Comment:
It doesn't matter at all, but the [bytes godoc](https://golang.org/pkg/bytes/#Buffer) seems to use `var buf bytes.Buffer` for some reason:

```go
package main

import (
	"bytes"
	"fmt"
	"os"
)

func main() {
	var b bytes.Buffer // A Buffer needs no initialization.
	b.Write([]byte("Hello "))
	fmt.Fprintf(&b, "world!")
	b.WriteTo(os.Stdout)
}
```

Reply:
Will change this everywhere in the follow ups.